SPECS/gdb/gdb.spec

```diff
@@ -1,6 +1,6 @@
 Summary:        C debugger
 Name:           gdb
-Version:        11.1
+Version:        11.2
```

Comment:
Since this is a version update with a new source tarball (gdb-11.2.tar.xz), please also update `gdb.signatures.json` with the updated sha256sum of the new 11.2 source tarball. This signatures file ensures the tarball consumed during the build matches what is expected.